Framework/Crystal/src/SortHKL.cpp

```diff
@@ -45,15 +45,25 @@ void SortHKL::init() {
   /* TODO: These two properties with string lists keep appearing -
    * Probably there should be a dedicated Property type or validator. */
   std::vector<std::string> pgOptions;
-  pgOptions.reserve(m_pointGroups.size());
+  pgOptions.reserve(2 * m_pointGroups.size() + 5);
+  for (auto &pointGroup : m_pointGroups)
+    pgOptions.push_back(pointGroup->getSymbol());
   for (auto &pointGroup : m_pointGroups)
     pgOptions.push_back(pointGroup->getName());
+  // Scripts may have Orthorhombic misspelled from past bug in PointGroupFactory
+  pgOptions.push_back("222 (Orthorombic)");
```

Comment:
Thanks. I think we should check for the misspelling and log a warning. We can then remove them in a release or 2.